WWCodeTokyo/speak-her-db
# Fix netlify lambda functions

The node-fetch 2.6.7 does not seems to be compatible with NodeJS [Abort Controller](https://developer.mozilla.org/en-US/docs/Web/API/AbortController). It broke the lambda function. 

Add a patch for the library for now. 

sample error message:
`TypeError - Expected signal to be an instanceof AbortSignal`

Resolves #262


**Proposed Changes**

This disable the mangle options of the webpack uglify plugin. 

**Testing Plan**

*How do the reviewers QA this? Feel free to write unit tests or describe manual testing steps here*

- Go to https://deploy-preview-278--speak-her-db.netlify.app/
- Click on "Find a speaker"
- see the tags
- :tada:
